app/src/main/java/com/keylesspalace/tusky/components/accountlist/AccountListFragment.kt

```diff
@@ -30,16 +30,16 @@ import androidx.recyclerview.widget.SimpleItemAnimator
 import autodispose2.androidx.lifecycle.AndroidLifecycleScopeProvider.from
 import autodispose2.autoDispose
 import com.google.android.material.snackbar.Snackbar
-import com.keylesspalace.tusky.AccountListActivity.Type
 import com.keylesspalace.tusky.BaseActivity
 import com.keylesspalace.tusky.R
-import com.keylesspalace.tusky.adapter.AccountAdapter
-import com.keylesspalace.tusky.adapter.BlocksAdapter
-import com.keylesspalace.tusky.adapter.FollowAdapter
-import com.keylesspalace.tusky.adapter.FollowRequestsAdapter
-import com.keylesspalace.tusky.adapter.FollowRequestsHeaderAdapter
-import com.keylesspalace.tusky.adapter.MutesAdapter
 import com.keylesspalace.tusky.components.account.AccountActivity
+import com.keylesspalace.tusky.components.accountlist.AccountListActivity.Type
```

Comment:
I've seen that the IDE changes such lists to com.keylesspalace.tusky.components.accountlist.adapter.*

Maybe that makes sense here?

Reply:
I think that is not allowed by our code style